tests/topotato/README

```diff
@@ -1,30 +1,52 @@
-this is WORK IN PROGRESS!
+TOPOTATO TESTING FRAMEWORK (WORK IN PROGRESS)
+==========================
 
-there are random pieces missing and other shit is still getting moved around
-and restructured!  don't make assumptions!
 
-It should be mostly working;  to run it you need:
+Installation
+============
 
-  apt-get install graphviz tshark tini python3-lxml python3-typing-extensions
-  sysctl -w kernel.unprivileged_userns_clone=1
-  mkdir /etc/frr
+Requirements
+- Python >= 3.8
 
-Also you need a patched pytest-html.
+Commands:
 
-After you set everything up, run:
+```sh
+sysctl -w kernel.unprivileged_userns_clone=1
+mkdir /etc/frr
 
-  ./run_userns.sh --frr-builddir=/path/to/frr/build --log-cli-level=DEBUG -v -v -x
+# In case you don't want to follow the <<apt install steps>> install these manually:
+# unshare - run program with some namespaces unshared from parent
+# nsenter - run program with namespaces of other processes
+# tini https://github.com/krallin/tini
+# dumpcap - Dump network traffic
+# ip - show / manipulate routing, network devices, interfaces and tunnels
 
-(you do NOT need to install FRR, it's executed from a source/build directory.)
-You can also put frr-builddir in pytest.ini (example included.)
+apt-get install graphviz tshark python3-venv wireshark-common
+wget https://github.com/krallin/tini/releases/download/v0.19.0/tini -O /bin/local/tini
+```
 
-REQUIREMENTS:
-- python >= 3.8 (3.6 used to work but not actively tested)
-- pytest >= 6.0
-- graphviz
-- tshark
-- dumpcap
-- tini
-- iproute2 >= 4.17
-- python lxml
-- typing-extensions (only for python 3.6 & 3.7, built-in for 3.8 and newer)
+Setup environments:
+===================
+
+```sh
+
+python3 -m venv .venv
```

Comment:
TBH I think using a `venv` is not advisable for topotato since it relies a lot on the system environment. Having an isolated python environment doesn't do much when the main issue is the network stack.

⇒ I would only add venv notes on the bottom in an extra section, in case the proper modules cannot be installed into the system.

Reply:
Agree, I added it as a footnote